BleManager.js

```diff
@@ -12,7 +12,7 @@ class BleManager  {
     return new Promise((fulfill, reject) => {
       bleManager.read(peripheralId, serviceUUID, characteristicUUID, (error, data) => {
         if (error) {
-          reject(error);
+          reject(new Error(error));
```

Comment:
Promise rejections will now have Error instances rather than String instances, this is for parity with the evothings library.